PhpAmqpLib/Wire/IO/SocketIO.php

```diff
@@ -25,14 +38,18 @@ class SocketIO extends AbstractIO
     /**
      * @param string $host
      * @param int $port
-     * @param float $timeout
+     * @param float $read_timeout
      * @param bool $keepalive
+     * @param float|null $write_timeout if null defaults to read timeout
+     * @param int $heartbeat
      */
-    public function __construct($host, $port, $timeout, $keepalive = false)
+    public function __construct($host, $port, $read_timeout, $keepalive = false, $write_timeout = null, $heartbeat)
```

Comment:
I meant a default value for backward compatibility. A value of 0/false/null that would mean no heartbeat.